### DISCUSSION TITLE
PNP Search Results Multiple Search Values

### DISCUSSION THREAD

**Discussion by zeannait** [microsoft-search/pnp-modern-search#2909] [General]
Using PNP Search Results, how do I pass multiple search values (separated by a comma) to Query Template in order to search on one Managed Property column?

**Comment by kasperbolarsen** [comment]
Are you looking for something like:

author:john* OR author:Patty*

**Comment by zeannait** [comment]
For example, in the PNP Search Box I can have something like this:

***@***.***

In this case I have two search values, but I can have more separated by a comma.

The URL that is sent to the PNP Search Results looks like this:  https://CompanyName.com/sites/SiteName/SitePages/SearchResults.aspx#water%2C%20Air

So, how would the Query Template separate the two (or more) search values?  The name of the Managed Property is called:  SearchValues.

**Reply by wobba** [reply]
author:({searchTerms})

There is no way to have two entered terms execute as an OR without the user specifying an OR. This is because by default there is an implicit AND between all parts of a query unless otherwise specified.

**Comment by zeannait** [comment]
Got it!  That worked.  Thank you.
